src/main/resources/assets/farmersdelight/lang/zh_tw.json

```diff
@@ -1,21 +1,23 @@
 {
   "block.farmersdelight.stove": "爐灶",
-  "block.farmersdelight.cooking_pot": "烹飪鍋",
+  "block.farmersdelight.cooking_pot": "燉鍋",
   "block.farmersdelight.basket": "籃子",
   "block.farmersdelight.cutting_board": "砧板",
-  "block.farmersdelight.oak_pantry": "橡木儲藏櫃",
-  "block.farmersdelight.birch_pantry": "樺木儲藏櫃",
-  "block.farmersdelight.spruce_pantry": "杉木儲藏櫃",
-  "block.farmersdelight.jungle_pantry": "叢林木儲藏櫃",
-  "block.farmersdelight.acacia_pantry": "相思木儲藏櫃",
-  "block.farmersdelight.dark_oak_pantry": "黑橡木儲藏櫃",
-  "block.farmersdelight.crimson_pantry": "緋紅蕈木儲藏櫃",
-  "block.farmersdelight.warped_pantry": "扭曲蕈木儲藏櫃",
+  "block.farmersdelight.skillet": "煎鍋",
+  "block.farmersdelight.oak_cabinet": "橡木儲藏櫃",
```

Comment:
Your lang is currently using keys from the 1.18.1 version, where some ID's changed. For 1.16.5, you'll need to revert them to the key that was previously there. Sorry for the confusion! 😅 

In this case, `cabinet` reverts to `pantry`.